components/list/list-item-mixin.js

```diff
@@ -689,7 +694,7 @@ export const ListItemMixin = superclass => class extends composeMixins(
 			'hide-bottom-border': this._showAddButton && (!this._hasNestedList || this._hasNestedListAddButton)
 		};
 
-		const alignNested = ((this.draggable && this.selectable) || (this.expandable && this.selectable && this.color) || (this.expandable && !this.selectable)) ? 'control' : undefined;
+		const alignNested = ((this.draggable && this.selectable) || (this.expandable && this.selectable && this.color) || (this._listItemInteractiveEnabled && this.expandable && !this.selectable)) ? 'control' : undefined;
```

Comment:
Flags the change [here](https://github.com/BrightspaceUI/core/pull/5551/files#diff-5bbbbcd3d2bf0b75e85e10abb2ad25850dd49abc2e3d716d6db33a764f361cf9R692)

I reviewed the rest of the css changes in that PR and they seemed safe (would not be triggered if flag is off)